ui/components/suite.jsx

```diff
@@ -1,27 +1,48 @@
-var React = require('react'),
-    Test = require('./test.jsx');
-
-var Suite = React.createClass({
-  statics: {
-    MARGIN_LEFT: 20,
-    FONT_SIZE: 100
-  },
-  render: function() {
-    var suite = this.props.suite,
+import React from 'react';
+import Test from './test.jsx';
+
+class Suite extends React.Component {
+
+  render() {
+    const suite = this.props.suite,
         suiteStyle = {
-          marginLeft: this.constructor.MARGIN_LEFT * (suite.indent - 1)
+          marginLeft: this.statics.MARGIN_LEFT * (suite.indent - 1)
         },
         titleStyle = {
-          fontSize: this.constructor.FONT_SIZE / suite.indent + '%'
+          fontSize: this.statics.FONT_SIZE / suite.indent + '%'
         };
+    let tests = [];
```

Comment:
This is still valid.

@adela-istrate-hs pushing commits while a review is still ongoing can make git hide some comments as being on an outdated diff. Please wait until the reviewer says "Finished review`"and after that you can push the commits upstream (meanwhile you can work locally).